pkg/catalog/catalog.go

```diff
@@ -64,7 +64,10 @@ func (mc *MeshCatalog) getAnnouncementChannels() []announcementChannel {
 	// See Github Issue: https://github.com/openservicemesh/osm/issues/1501
 	go func() {
 		ticker := time.NewTicker(updateAtLeastEvery)
-		ticking <- ticker.C
+		for {
+			<-ticker.C
+			ticking <- nil
+		}
```

Comment:
A comment here will help.

Moreover, since this change is fixing a bug, a unit test to catch this will help as well.

Reply:
We started more elaborate work on this here: https://github.com/openservicemesh/osm/issues/1501

I propose we patch the immediate single-tick issue with this PR; then continue the work on this via https://github.com/openservicemesh/osm/issues/1501

Overall I think we should overhaul this ticker to become a debug feature;  When something is not working and we need to force config - we could enable it; I can imagine situations in which it would be beneficial to have a force-refresh (once or on an interval)